src/cplaylist.c

```diff
@@ -178,6 +179,52 @@ mpd_run_save(struct mpd_connection *connection, const char *name)
 		mpd_response_finish(connection);
 }
 
+enum mpd_queue_save_mode mpd_parse_queue_save_mode(const char *p)
+{
+	if (strcmp(p, "create") == 0)
+		return MPD_QUEUE_SAVE_MODE_CREATE;
+	else if (strcmp(p, "replace") == 0)
+		return MPD_QUEUE_SAVE_MODE_REPLACE;
+	else if (strcmp(p, "append") == 0)
+		return MPD_QUEUE_SAVE_MODE_APPEND;
+	else
+		return MPD_QUEUE_SAVE_MODE_UNKNOWN;
+}
+
+const char *
+mpd_lookup_queue_save_mode(enum mpd_queue_save_mode mode)
+{
+	switch (mode) {
+	case MPD_QUEUE_SAVE_MODE_CREATE:
+		return "create";
+	case MPD_QUEUE_SAVE_MODE_REPLACE:
+		return "replace";
+	case MPD_QUEUE_SAVE_MODE_APPEND:
+		return "append";
+	default:
```

Comment:
This `default:` disables the compiler warning when a new enum gets added, but we shouldn't suppress that warning

Reply:
Correct, I have not taken this into account. Fixed in the commit.